skills/declarative/calendarSkill/language-generation/en-us/event.en-us.lg

```diff
@@ -259,7 +259,9 @@
     -
 
 # EventAttendeeContainer(event)
-- IF: ${ event.attendees != null && count(event.attendees) > 1 }
+- IF: ${ event.Response == 'organizer' }  
```

Comment:
New person question - can we just eliminate this if and instead have 

"IF ${event.Response != 'organizer' && event.attendees != null && count(event.attendees) > 1}"

Maybe it's clearer that way

Reply:
I don't think this logic is quite right. If the user is the organizer, this will prevent it from showing any of the other attendees, when really we only want to hide the attendees if the user is the organizer and there are no other people coming to the meeting.

So the condition that checks for count(event.Attendees) == 1 should also include "&& event.Response != 'organizer"'.

Reply:
Actually, the previous logic is enough for most cases except a special one, that is the event.Attendees sometimes include the orgnizer him/herself and sometimes not include. So even if count(event.Attendees) ==1, we couldn't know whether the attendee is the orgnizer him/herself.